.ci/install_rust.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Ant Financial
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cidir=$(dirname "$0")
+rustarch=$(uname -m)
+release="nightly"
+
+if [ "${rustarch}" == "ppc64le" ]; then
+	rustarch="powerpc64le"
+fi
+
+if ! command -v rustup > /dev/null; then
+	curl https://sh.rustup.rs -sSf | sh
+fi
+
+# sudo -E apt-get update
+# sudo -E apt-get -y install build-essential g++ make cmake automake autoconf m4 libc6-dev libstdc++-8-dev coreutils binutils debianutils gcc musl musl-dev musl-tools git
+
+rustup toolchain install ${release}-${rustarch}-unknown-linux-gnu
+rustup default ${release}-${rustarch}-unknown-linux-gnu
```

Comment:
You might want to add a check for `KATA_DEV_MODE` to avoid surprising people running this "outside of the CI":

- https://github.com/kata-containers/tests/#developer-mode

Reply:
fixed